tests/postgres_tests/test_bulk_update.py

```diff
@@ -5,13 +5,13 @@
     HStoreModel, IntegerArrayModel, NestedIntegerArrayModel,
     NullableIntegerArrayModel, OtherTypesArrayModel, RangesModel,
 )
-
+from django.test import modify_settings
 try:
     from psycopg2.extras import NumericRange, DateRange
 except ImportError:
     pass  # psycopg2 isn't installed.
 
-
+@modify_settings(INSTALLED_APPS={'append': 'django.contrib.postgres'})
```

Comment:
Can you not apply this decorator up in `PostgreSQLTestCase` instead?

Reply:
It led to a couple of unexpected errors and test failures. We could subclass PostgreSQLTestCase and apply the decorator on the subclass. That was something I wanted to do but forgot it in my backlog.